doc_src/index.hdr.in

```diff
@@ -17,7 +17,201 @@
+- add `/usr/local/bin/fish`  to  `/etc/shells`
+- change your default shell with `chsh -s` to `/usr/local/bin/fish`
+
+For for detailed instructions see <a href="tutorial.html#tut_switching_to_fish">Switching to *fish*</a>.
+
+\subsection uninstall Uninstalling
+
+For uninstalling *fish*: see <a href="faq.html#faq-uninstalling">FAQ: Uninstalling *fish*</a>
+
+\subsection start Starting and Exiting
+
+Once *fish* has been installed, open a terminal. If *fish* is not the default shell:
+
+- Enter `fish` to start a *fish* shell:
+
+\fish
+>fish
+\endfish
+
+- Enter `exit` to exit a *fish* shell:
+
+\fish
+>exit
+\endfish
+
+\subsection bash Executing Bash
+
+If *fish* is your default shell you can still execute *bash* commands in case you need to:
+
+- a single command can be executed with:
+
+\fish
+>bash -c SomeBashCommand
+\endfish
+
+- or a *bash* shell can be opened with:
+
+\fish
+>/bin/bash
+\endfish
+
+This might be useful, when copying complicated *bash* commands from a website. Translating them into *fish* syntax can be avoided that way.
+
+<a href="#install"><-</a><hr>
+
+\section shell What is a Shell
+
+This section is about the basics of shells. *fish* is a shell and shells have a lot in common:
+
+- <a href="#shell-standards">Shell Standards</a>: Why shells adjust to standards
+- <a href="#man-page">Manual Pages</a>: Commands usually come with a standardized manual page
+- <a href="#shell-syntax">Command Syntax</a>: Shell commands have a standard syntax
+- <a href="#shell-programming">Commands versus Programs</a>: Commands differ from normal programs
+- <a href="#shebang-line">Shebang Line</a>: How the shell knows the language of a script
+
+\subsection shell-standards Shell Standards
+
+A shell is an interface to the operating system that reads from the commandline of a terminal. A shell's task is to identify and interpret commands. The commands can come from different applications and can be written in different programming languages.
+
+This can only work smoothly if shells adapt to some common standards. For shells there is the POSIX standard, see <a href="https://en.wikipedia.org/wiki/Command-line_interface#Command-line_interpreter">Command-line interpreters</a>. `fish` tries to satisfy the POSIX standard wherever it does not get into the way of its own design principles, see <a href="design.html">`fish` Design</a>.
+
+\subsection man-page Manual Pages
+
+There is a common standard on how to receive help on shell commands: applications provide a manual page to their commands that can be opened with the `man` command:
+
+\fish
+>man COMMAND
+\endfish
+
+This convention helps to make sure help can be found on commands no matter where they originate from. *fish*'s internal commands all come with a manual page.
+
+\subsection shell-syntax Command Syntax
+
+Shell commands also have some syntax in common, no matter where they originate from: 
+
+Commands consist of three parts:
+
+\fish
+COMMAND [OPTIONS] ARGUMENTS
```

Comment:
Any way to explain that options can (most of the time nowadays) also appear after arguments? I.e. that the ordering really is not that strict?

Or is that too much this early on?

Reply:
I felt that was to early to introduce several options to write commands here. But I wrote now that this is just one basic pattern.